File: internal/graph/testdata/java_simple/com/example/Handler.java
package com.example;

public class Handler {
    private String name;

    public Handler(String name) {
        this.name = name;
    }

    public String greet() {
        return this.format(this.name);
    }

    public String format(String x) {
        return "Hello, " + x;
    }
}

File: internal/graph/testdata/js_simple/src/handler.js
export class Handler {
  constructor(name) {
    this.name = name;
  }

  greet() {
    return this.format(this.name);
  }

  format(x) {
    return "Hello, " + x;
  }
}

export function makeHandler(name) {
  return new Handler(name);
}
